db/migrate/20190630034751_create_host_facets_reported_data_facets.rb

```diff
@@ -0,0 +1,10 @@
+class CreateHostFacetsReportedDataFacets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :host_facets_reported_data_facets do |t|
+      t.references :host_id, type: :integer, foreign_key: true, index: { unique: true }
```

Comment:
:+1: I like we managed to solve it :relaxed:

Reply:
I had to change `host_id` to `host` though, migrations would fail, I'll push the fix with other updates